tools/wasm_builder_cli/README.md

```diff
@@ -1,23 +1,23 @@
-# `iroha_wasm_builder_cli`
+# `wasm_builder`
 
-A CLI around [`iroha_wasm_builder`](../wasm_builder) crate.
+A CLI for building wasm smartcontracts.
 
 ## Usage
 
 **Check the smartcontract:**
 
 ```bash
-iroha_wasm_builder_cli check path/to/project
+wasm_builder check path/to/project
```

Comment:
I do not think it's a good idea to remove iroha from the name, seeing as this is an iroha-specific development tool that the user might want to install to their `PATH`. `iroha_wasm_builder` or `iroha-wasm-builder` would be preferrable (though the `cli` is definitely not useful in the executable name)

Reply:
fixed